packages/components/ContextualMenu/src/ContextualMenuItem.tsx

```diff
@@ -62,7 +67,29 @@ export const ContextualMenuItem = compose<ContextualMenuItemType>({
       icon: !!icon
     };
 
-    const cmRef = useViewCommandFocus(userProps.componentRef);
+    const onKeyUp = React.useCallback(
+      e => {
+        if ((e.nativeEvent.key === 'Enter' || e.nativeEvent.key === ' ') && pressable.state.focused) {
+          onItemClick(e);
+        }
+      }, [onItemClick, pressable.state]);
+
+    const onMouseEnter = React.useCallback(
+      e => {
+        pressable.props.onMouseEnter && pressable.props.onMouseEnter(e);
+        pressable.props.onFocus && pressable.props.onFocus(e);
```

Comment:
onFocus should get called automatically by the call to focus() above.

Honestly I think the 

...pressable.props 

below is already providing an onMouseEnter/Leave that should work, you shouldn't need to override the implementation from the pressable hook, so I think you can just delete your onMouseEnter/Leave callbacks

Reply:
I initially tried implementing without these but without calling pressable.props.onFocus()/onBlur(), mouse focus stays on the last hovered item when I hover off the menu.

Reply:
You mean visually?

Can you just drive the visuals off hovered state instead of the focus state then? (By moving the tokens you're currently setting in settings for focused to the hovered override) ?

Reply:
It doesn't seem to be a visual only issue. The main issue is when the mouse hovers off the menu, actual focus remains on the last hovered/focused menu item. When I press space bar or enter, the menu item gets invoked.